app/models/header.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const Header = function (data, db) {
+  this.data = data;
+  this.db = db;
+};
+
+Header.prototype.get = (name) => {
```

Comment:
@rodrigopavezi This shouldn't use `=>` here, it should be `function (name) {`. I'm getting an error when trying to use the `get()` function. Cheers